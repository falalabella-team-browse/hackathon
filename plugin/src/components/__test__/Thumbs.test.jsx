import React from "react";
import { shallow } from "enzyme";
import Thumbs from "../Thumbs";

describe("Thumbs", () => {
  it("should render properly, with props", () => {
    const wrapper = shallow(<Thumbs options={{ color: "#fff" }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
