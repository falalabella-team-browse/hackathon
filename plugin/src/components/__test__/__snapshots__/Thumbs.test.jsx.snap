// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbs should render properly, with props 1`] = `ShallowWrapper {}`;
